src/main/java/com/marklogic/contentpump/AggregateXMLReader.java

```diff
@@ -133,7 +133,7 @@ protected void initStreamReader(InputSplit inSplit) throws IOException,
         try {
             xmlSR = f.createXMLStreamReader(fInputStream, encoding);
         } catch (XMLStreamException e) {
-            LOG.error(e.getMessage(), e);
+            LOG.error("Parse error", e);
```

Comment:
Maybe it makes sense to write `LOG.error("Parse error: ", e)` to make the logs more readable? 

Please make sure to update the keys of the corresponding tests. :)

Reply:
I agree with the suggestion for ":". And I think we need to discuss more on where to place this warning as this error is rewritten to match the test key. It would be better to edit this error message and test key as the parser is yet to parse on the object at this point in the code. It is in the process of creating a proper XML reader.

Reply:
Changed the error to previous state. The change will be made in the test key. Key would match word "invalid byte"